bin/schematic.py

```diff
@@ -107,3 +109,5 @@ def count_col(col):
     # obviously you can only build from within minecraft
     for slice in schematic:
         build_slice(slice, block_defs)
+    navigator.go_to([0,0,0])
```

Comment:
Your navigator knows how to get back to 0,0,0 but it has no guarantee that it's path won't be obstructed.

This line looks like the path would just cut straight through the building that was just created.